node_modules/oae-core/preferences/preferences.html

```diff
@@ -111,7 +111,7 @@ <h3 id="preferences-modal-title">__MSG__MY_PREFERENCES__</h3>
                             <i class="fa fa-check hide"></i>
                             <div class="well oae-large-options">
                                 <input type="radio" id="preferences-email-immediate" value="immediate" name="preferences-email-group" class="pull-left" tabindex="0" {if oae.data.me.emailPreference === 'immediate'} checked{/if}/>
-                                <i class="fa fa-bolt large text-center {if oae.data.me.emailPreference === 'immediate'} selected{/if}"></i>
```

Comment:
Can you verify that removing the selected styling is not causing any visual regressions in those areas either?